src/pages/home/ReportScreen.js

```diff
@@ -295,8 +297,8 @@ class ReportScreen extends React.Component {
                                     />
                                 )}
                             <ReportFooter
-                                addWorkspaceRoomErrors={addWorkspaceRoomErrors}
-                                addWorkspaceRoomPendingAction={addWorkspaceRoomPendingAction}
+                                errors={addWorkspaceRoomOrChatErrors}
```

Comment:
ReportFooter does indeed take `addWorkspaceRoomErrors` and `addWorkspaceRoomPendingAction` as props - https://github.com/Expensify/App/blob/7044ab2b96067081c8f67191e51fa859dba66d91/src/pages/home/report/ReportFooter.js#L35-L40